titanium/package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "titanium",
-  "version": "1.4.0",
+  "version": "2.0.0",
   "implName": "Titanium",
   "contributors": [{
     "name": "Chris Price",
     "username": "chrisprice",
     "hash": "b6fe36176ec7c9475374a5cd3b7bef1a"
+  }, {
+    "name": "Fokke Zandbergen",
+    "username": "fokkezb"
```

Comment:
if you add your [gravatar hash](https://en.gravatar.com/site/implement/hash/) it means we can add you gravatar image to the website, if you like.